cumulusci/tasks/github/util.py

```diff
@@ -53,15 +53,15 @@ def __call__(
         self.local_dir, self.repo_dir = self._validate_dirs(local_dir, repo_dir)
         self._set_git_data(branch)
 
-        new_tree_list = [self._create_new_tree_item(item) for item in self.tree]
-        self._add_new_files_to_tree(new_tree_list)
+        self.new_tree_list = [self._create_new_tree_item(item) for item in self.tree]
```

Comment:
Is there a benefit to having this as a member variable rather than a standalone variable? My initial though would be that we may want to reference this elsewhere in a different task, but I'm not sure on the context.

Reply:
@Br4nd0R this change was for the test, so that it can easily verify that the correct new tree was produced.